content/200-concepts/300-more/300-telemetry.mdx

```diff
@@ -6,18 +6,18 @@ metaDescription: 'This page explains how Prisma uses telemetry features to impro
 
 ## Overview
 
-The term **telemetry** refers to the collection of certain usage data to help _improve the quality of a piece of software_. Prisma 2 uses telemetry in two contexts:
+The term **telemetry** refers to the collection of certain usage data to help _improve the quality of a piece of software_. Prisma uses telemetry in two contexts:
 
-- collecting usage data
-- submitting error reports
+- collecting CLI usage data
```

Comment:
```suggestion
- when it collects CLI usage data
```

Reply:
If you want to get rid of the `...ing`, I think the better option here would be to say:
```suggestion
- to collect CLI usage data
```